graylog2-web-interface/src/components/streams/StreamMetaData.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+import CollapsibleStreamRuleList from 'components/streamrules/CollapsibleStreamRuleList';
+import { Pluralize } from 'components/common';
+import { Button } from 'components/graylog';
+
+import StreamThroughput from './StreamThroughput';
+
+const StreamMetaData = ({ isDefaultStream, stream, streamRuleTypes, permissions }) => {
+  const [expanded, setExpanded] = useState(false);
+  const toggleText = expanded ? 'Hide' : 'Show';
+
+  const _formatNumberOfStreamRules = () => {
+    if (stream.is_default) {
+      return 'The default stream contains all messages.';
+    }
+    if (stream.rules.length === 0) {
+      return 'No configured rules.';
+    }
+
+    let verbalMatchingType;
+    switch (stream.matching_type) {
+      case 'OR': verbalMatchingType = 'at least one'; break;
+      default:
+      case 'AND': verbalMatchingType = 'all'; break;
+    }
+
+    return (
+      <span>
+        Must match {verbalMatchingType} of the {stream.rules.length} configured stream{' '}
+        <Pluralize value={stream.rules.length} plural="rules" singular="rule" />.
+      </span>
+    );
+  };
+
+  const _onHandleToggle = () => {
+    setExpanded(!expanded);
+  };
+
+  return (
+    <div className="stream-metadata">
+      <StreamThroughput streamId={stream.id} />. {_formatNumberOfStreamRules()}
+
+      {!isDefaultStream && (
+        <>
+          <span className="stream-rules-link">
+            <Button bsStyle="link"
```

Comment:
Small thing: using an actual button here makes sense, but breaks the link alignment with text, since the browser will render it differently as a regular anchor:

<img width="534" alt="Screenshot 2020-01-30 at 11 44 54" src="https://user-images.githubusercontent.com/716185/73443151-73f66f00-4356-11ea-9a1b-992dc293e193.png">

Reply:
![image](https://user-images.githubusercontent.com/122591/73558283-77eac580-4418-11ea-8a4e-843c1239e241.png)